patches/graceful-fs+4.2.3.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/node_modules/graceful-fs/graceful-fs.js b/node_modules/graceful-fs/graceful-fs.js
+index 8c75ee2..df8ff2e 100644
+--- a/node_modules/graceful-fs/graceful-fs.js
++++ b/node_modules/graceful-fs/graceful-fs.js
+@@ -41,42 +41,6 @@ if (!global[gracefulQueue]) {
+     }
+   })
+ 
+-  // Patch fs.close/closeSync to shared queue version, because we need
+-  // to retry() whenever a close happens *anywhere* in the program.
+-  // This is essential when multiple graceful-fs instances are
+-  // in play at the same time.
+-  fs.close = (function (fs$close) {
```

Comment:
This seems safe as long as we're only using one instance of `graceful-fs` in our build system. Which is true at the moment. I guess we'll just have to keep an eye on this going forward.

Reply:
previously you could not modify the exports of other packages (here its `fs`), but Lavamoat no longer protects module exports. this patch can be removed.

Reply:
this patch should be removed